Down-Example/Down-Example/ViewController.swift

```diff
@@ -11,66 +11,67 @@ import Down
 
 class ViewController: UIViewController {
 
+    @IBOutlet var activityIndicatorView: UIActivityIndicatorView!
+    @IBOutlet var visualEffectContentView: UIView!
+    var downView: DownView!
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        
-        guard let downView = try? DownView(frame: self.view.bounds, markdownString: markdownString, didLoadSuccessfully: {
-            // Optional callback for loading finished
-            print("Markdown was rendered.")
-        }) else { return }
-        view.addSubview(downView)
+
+        // Note: No errors will be thrown here as we're loading an empty Markdown string.
+        downView = try! DownView(frame: view.bounds, markdownString: "")
+        downView.isHidden = true
+        visualEffectContentView.addSubview(downView)
+
+        createStatusBarBackgrounds()
+
+        DispatchQueue.global(qos: .userInteractive).async {
+            self.loadReadMe()
+        }
+    }
+
+    func loadReadMe() {
+        var readMeContents: String
+
+        do {
+            let readMeURL = URL(string: "https://raw.githubusercontent.com/iwasrobbed/Down/master/README.md")!
```

Comment:
I would prefer to render local markdown here (so people don't think it's slow based upon their network connection). 

It looks like that was your first intention with `README-11:2:18.md` but I don't see it being used. Mind switching back to that?

The `progressIndicator` won't be necessary in that scenario.

Reply:
Ah, good catch. Both now use a shared local version.